lib/soapy_cake/response.rb

```diff
@@ -22,6 +22,14 @@ def to_enum
       end
     end
 
+    def to_xml
```

Comment:
Won't work, since when batching the XML body will never be empty and `AdminBatched` won't stop attempting to request the next batch.